types/react-native/index.d.ts

```diff
@@ -5967,12 +5971,29 @@ interface PlatformStatic {
 }
 
 interface PlatformIOSStatic extends PlatformStatic {
+    constants: PlatformConstants & {
+        forceTouchAvailable: boolean;
+        interfaceIdiom: string;
+        osVersion: string;
+        systemName: string;
+    };
     OS: 'ios';
     isPad: boolean;
     isTVOS: boolean;
 }
 
 interface PlatformAndroidStatic extends PlatformStatic {
+    constants: PlatformConstants & {
+        Version: number;
+        Release: string;
+        Serial: string;
+        Fingerprint: string;
+        Model: string;
+        Brand: string;
+        Manufacturer: string;
+        ServerHost?: string;
+        uiMode: string;
```

Comment:
I would propose to make this more specific:
```suggestion
        uiMode: 'car' | 'desk' | 'normal' | 'tv' | 'watch' | 'unknown';
```

Reply:
Makes sense.